openfisca_uk/variables/finance/benefit/family/universal_credit.py

```diff
@@ -506,7 +506,18 @@ def formula(benunit, period, parameters):
             "self_employment_income",
             "miscellaneous_income",
         ]
-        earned_income = aggr(benunit, period, INCOME_COMPONENTS)
+        personal_earned_income = add(
```

Comment:
Does `add` aggregate over `benunit.members`? Trying to parse what this section is doing. A comment or two would be helpful.

Would this be clearer if some of it was in a person-level variable?

Reply:
I'll add some comments - but I think the Minimum Income Floor is on a per-person basis, with couples' MIF floors being added together (but only if they are both self-employed). This does a lot of the person-level calculation, before summing to the benefit unit level.